cbt.go

```diff
@@ -1181,6 +1183,19 @@ func doLookup(ctx context.Context, args ...string) {
 		filters = append(filters, columnFilters)
 	}
 
+	keysOnly := false
+	if keyStr := parsed["keys-only"]; keyStr != "" {
+		keysOnly, err = strconv.ParseBool(keyStr)
+	}
+	if err != nil {
```

Comment:
Could this move up to be directly under `strconv.ParseBool`? It's a little odd as written.

Reply:
I was trying to avoid a nested if statement. If it looks more odd like this, then I'll move it.